### PR TITLE
SILGen: Don’t make class accessors transparent.

### PR DESCRIPTION
**Pull request by eeckstein**
Because when they are inlined they might access hidden symbols in another module, like the field offset variable.

fixes rdar://problem/29707641

@slavapestov Thanks for your help!

### CODE REVIEW

**Comment by eeckstein** [comment]
@swift-ci Please smoke test and merge

**Comment by eeckstein** [comment]
@swift-ci Please test

**Comment by eeckstein** [comment]
@jckarter, @slavapestov  Can you please take a look at that?

File: test/multifile/synthesized-accessors/two-modules/library.swift [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // RUN: true
 
+import Foundation
```
Comment (slavapestov) [inline test/multifile/synthesized-accessors/two-modules/library.swift]: We want to make tests cross platform where possible. Can you split off a new test if you want to test NSObject subclasses?

**Comment by eeckstein** [comment]
@swift-ci Please test

**Comment by eeckstein** [comment]
@swift-ci Please test

File: test/multifile/synthesized-accessors/two-modules/library.swift (hunk 1 above)
Comment (eeckstein) [inline test/multifile/synthesized-accessors/two-modules/library.swift]: I added a #if _runtime(_ObjC)

**Comment by slavapestov** [comment]
Looks good, thanks for fixing this!

**Pull request merged**